flare/ase/calculator.py

```diff
@@ -28,30 +28,30 @@ def __init__(self, gp_model, mgp_model=None, par=False, use_mapping=False):
         self.par = par
         self.results = {}
 
-    def get_property(self, atoms, property_name):
-        if property_name not in self.results.keys():
+    def get_property(self, name, atoms=None, allow_calculation=True):
+        if name not in self.results.keys():
```

Comment:
This PR seems to fix my issue (thanks!) but makes the code run insanely slowly compared to having ``trajectory=None`` in my tests. I think ``allow_calculation`` should be actually handled here [the way ASE does it](https://gitlab.com/ase/ase/-/blob/f9c6203b03620d4231c1c147c170c9196a1f14d1/ase/io/trajectory.py#L427-429):

```suggestion
        if name not in self.results.keys():
            if not allow_calculation:
                return None
```

⇒ Fast again

Reply:
Thanks for finding this issue and giving the suggestion. A new commit has been made on this `allow_calculation` change.